llvm/include/llvm/CodeGen/GlobalISel/MachineIRBuilder.h

```diff
@@ -383,9 +386,15 @@ class MachineIRBuilder {
   /// Set the PC sections metadata to \p MD for all the next build instructions.
   void setPCSections(MDNode *MD) { State.PCSections = MD; }
 
+  /// Set the PC sections metadata to \p MD for all the next build instructions.
+  void setMMRAMetadata(MDNode *MMRA) { State.MMRA = MMRA; }
```

Comment:
I would expect get/set methods to be arranged together for each property. For example, the ones for DebugLoc just above.